src/BackendPkg/User.go

```diff
@@ -11,17 +11,10 @@ type User struct{
 	Email string `json:"Email"`
     UserName string `json:"UserName"`
```

Comment:
I guess another thing here and in our tables. Is it UserName or Username with camel case?

Reply:
We  did "UserName" in the tables